components/clp-py-utils/clp_py_utils/clp_config.py

```diff
@@ -24,6 +24,7 @@
 COMPRESSION_WORKER_COMPONENT_NAME = "compression_worker"
 SEARCH_WORKER_COMPONENT_NAME = "search_worker"
 WEBUI_COMPONENT_NAME = "webui"
+CONTROLLER_COMPONENT_NAME = "controller"
```

Comment:
* Do we need to add this here since we're only using it in `start-clp.py`? Do you plan to add it to `stop-clp.py`?
* How about `CONTROLLER_COMPONENTS_TARGET` to differentiate it from the actual component names?